kaitaistruct.py

```diff
@@ -279,57 +351,67 @@ def read_bits_int_le(self, n):
     # Byte arrays
     # ========================================================================
 
-    def read_bytes(self, n):
+    def alignment(self, a):
```

Comment:
needs docs

Reply:
What format and where should I put it?

Reply:
It is currently not very what it does and how to use it and why it is implemented this way and not another one.

Reply:
It's addressing this: https://github.com/kaitai-io/kaitai_struct/issues/12#issuecomment-277757932 Some structures need alignment to be read (like Unity serialized files for example). It's convenient to have this ready for such cases and maybe for expanding ksc format with this at some point.